### DISCUSSION TITLE
ガイドラインについて改善要望

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#231] [要望・アイデア・不具合]
Qiitaの記事の質を上げるため施策として、ガイドラインの改善要望を下記に記載します。

前提としてQiitaの**行動指針**の一つとして「**エンジニアにとって再利用性・汎用性の高い情報が集まる場をつくろう**」を実現及び維持するためには、互いに有益である情報を共有し続けるということが本質だと思っています。

- ガイドラインの内容改善
→既に同様の意見はあがっていると思いますが、読書、感想文、ポエムでタグ付けされているような記事を無くしていく取り組みが必要だと思います。ガイドラインもそのための一つの施策だと思っていますので、改めてあるべき姿を再定義し、読書、感想文、ポエムでタグ付けされているような記事を排除するための施策の検討をお願いします。

　　客観的に見ても読書、感想文などでタグ付けされた記事は二時情報でしかなく、その人の読んだ感想が「エンジニアにとって再利用性・汎用性の高い情報」として解釈するのは厳しいと思います。また、著作権侵害にも触れてしまう可能性も生みやすいため、そのような可能性を与える機会を作りやすくしていることはプラットフォームとしても良くはないのではないでしょうか。

　　恐縮ですが現在のガイドラインについて方向性の再定義をした方がこれからの発展にも繋がると思いますので、ご検討のほどお願いします。例えばですが、技術書を読んだ感想について語りありたいのであれば、「質問」のように別フィールドを作る検討をされてはいかがでしょうか。

- ガイドラインの位置修正
→現在はフッタの「ガイドとヘルプ」の下に記載がありますが、記事を書くユーザーから見て場所が分かりにくいと思います。
　ユーザーの動きとして意識してガイドラインを確認する人もいれば、ガイドラインを全く見ないで記事を書く人も大勢いると思います。現在の位置の場合、意識してガイドラインを確認する人に対してはリーチが届きににく、全く見ないで書く人に対しては効果が薄いと考えています。
    
　　従ってHOME画面のガイドラインの位置をフッタに変更及び、記事作成時の[画面](https://qiita.com/drafts/new)からでも目に入る位置に修正した方が良いと思いました。参考になれば幸いです。

**Comment by tanaka-asami415** [comment]
@Brutus03 

貴重なご意見をいただきありがとうございます🙏

>読書、感想文、ポエムでタグ付けされているような記事を無くしていく取り組みが必要だといます。

2021年4月にガイドラインを改定し、エンジニアにとって「役立つ」ような内容であれば、
様々な内容の投稿をしていただけるようになりました。
↓詳細はこちらをご参照ください。
https://blog.qiita.com/notice-community-guideline-update/

そのため、運営側として排除することはせず、ユーザーさまの判断に委ねております。

ただ、今回のご要望とは違う形にはなりますが、#158 でいただいたタグミュート機能のような形で、
自分が興味のある分野だけを見れるような改善は考えております🙇


>著作権侵害にも触れてしまう可能性も生みやすいため

この点はおっしゃる通りで、通報機能もありますが
より的確に対応できるような仕組み作りは今後の課題です。


>技術書を読んだ感想について語りありたいのであれば、「質問」のように別フィールドを作る検討をされてはいかがでしょうか。

現在も質問機能で意見交換ができるので、そちらを活用していただけるようなアナウンスはできるかもしれません🙏

>ガイドラインの位置修正

こちらですが、記事作成時の「本文」タブの右側に「良い記事を書くには」というガイドがあるのと
投稿時のモーダルに「コミュニティガイドライン」の確認が出るようになっているのですが、これだけだと不十分でしょうか？🙏
それとも視認性が低く、役目を果たせていないでしょうか？🙇

<img width="586" alt="投稿時のモダール" src="https://user-images.githubusercontent.com/103916185/170461188-d6fd8098-ec96-4edf-bf3b-1958751185a8.png">

**Comment by Brutus03** [comment]
@tanaka-asami415 
ご回答ありがとうございます。

> そのため、運営側として排除することはせず、ユーザーさまの判断に委ねております。

運営側の考え方について承知しました。

>ただ、今回のご要望とは違う形にはなりますが、[#158](https://github.com/increments/qiita-discussions/discussions/158) でいただいたタグミュート機能のような形で、 自分が興味のある分野だけを見れるような改善は考えております🙇

私も感覚値になりますが、ここ数年特に最近の傾向として読書、感想文、ポエムのタグ付けが増えていると感じ、今回意見させていただきました。

例えばですが、Yahooでニュースを見ようとしてニューストピックが、ポエムだったらユーザーは見ないと思います。同じように私達エンジニアは技術に関する記事を見る目的でQiitaを利用しています。恐縮ですがホームやトレンドの画面にこのようなタグ付けされた記事を見せられることに、運営サイドが思っている以上にユーザーからすると非常にストレスを感じることについてご理解いただけると幸いです。

なお、アプローチとして記事を書くという行為の制御は現実的ではなく、そこまでの対応は求めていませんが、せめて運営側の仕組みとしてホームやトレンドの画面に表示される記事については読書、感想文、ポエムなどのタグを除外するなどの検討をされてはどうでしょうか。理想としてはユーザーにタグミュート機能のような行為を行わせるのではなく、運営側で制御するのがあるべき姿だと思います。

>こちらですが、記事作成時の「本文」タブの右側に「良い記事を書くには」というガイドがあるのと 投稿時のモーダルに「コミュニティガイドライン」の確認が出るようになっているのですが、これだけだと不十分でしょうか？

1ユーザーの意見としては不十分だと感じています。記事を書ききった最後に確認するユーザーは少ないと思います。案としては赤字などで目につく形で装飾してフッタに組み込んだ方が注意を引きやすいと思います。視認性についてはアンケートをとるなど、リアルな数字を得るのがいいと思います。

要約すると現在のガイドラインについてはまだ改善の余地を感じています。全体的にユーザーに伝えたい内容がフワッとしているため、運営側でガバナンスを効かせた方がいいと思います。恐れ入りますが「Qiitaで投稿するべきではない記事」について見直していただけると幸いです。ご検討のほどお願いします。

**Reply by tanaka-asami415** [reply]
@Brutus03

非常に建設的なご意見をいただき、改めてありがとうございます🙇

基本的に今回いただいた内容は検討していきます。
興味の無い記事が多く流れてくる体験をさせてしまっていることは、運営としても強く課題に感じています。

ただ、

- Qiitaとしては、ユーザーの裾野を広げていきたいという思いがある
- 今回挙げていただいたような「投稿すべきでない記事」に価値を感じているユーザーも多くいらっしゃる

このような理由から一律で禁止するというよりは、
タグのミュートや、フィードを技術とそれ以外でわけるなどの方法で、受け取りたい情報をそれぞれの人が選ぶ世界にしたいと思っています。

今回いただいたご意見をすべて解決することができず申し訳ないのですが、ご理解いただけますと幸いです🙇

ガイドラインの視認性に関しましても、ありがとうございます🙏
場所やデザインなど、検討させていただきます🙇

**Comment by Brutus03** [comment]
@tanaka-asami415 
>興味の無い記事が多く流れてくる体験をさせてしまっていることは、運営としても強く課題に感じています。

念の為補足になりますが興味の無い記事が多く流れてくる体験というより、そもそものQiitaとしての前提が崩壊していることに対して危機感や失望を感じています。

全体の母数として技術記事があってその中でユーザーが必要な情報を取捨選択するのが本質だと思います。しかし、今のQiitaは技術記事以外にポエムや読書感想文などのノイズが大きくなることで、必要な情報を見つけにくく、セレンディピティ以前にストレスを与える要因が強くなっていることを課題に感じています。

例えるならスニーカーを買おうと思って靴屋に行ったら、いきなり食料品などが置いてあるイメージです。

現在Zennなど他社サービスも存在し、そちらに記事が流れているのも見て取れますし、このまま行くと技術記事はZennでポエムなどはQiitaといったような利用をされる場合、サービスの価値も廃れていくのではないか危惧しています。

>今回いただいたご意見をすべて解決することができず申し訳ないのですが、ご理解いただけますと幸いです🙇

承知しました。理解します。

>ガイドラインの視認性に関しましても、ありがとうございます🙏 場所やデザインなど、検討させていただきます🙇

是非、ご検討のほどお願いします。

総括ですが今回の意見について運営側にて再度、課題定義していただき、引き続き施策の検討をお願いします。
また、検討した内容を適宜ユーザーにフィードバックしていただけると幸いです。

以上です。

**Reply by tanaka-asami415** [reply]
@Brutus03 
ありがとうございます🙏

今回頂いたご意見を真摯に受け止め、今後の改善に努めてまいります。
お時間はかかってしまうかもしれませんが、何かしらお知らせできる様になった時には、
ブログやDiscussions等でしっかりとアナウンスやフィードバックをさせていただきます🙇
